### DISCUSSION TITLE
URGENT: Potential Risk Found in "Fan Control" Software

### DISCUSSION THREAD

**Discussion by CaptainCurry93** [Rem0o/FanControl.Releases#1836] [General]
Based on my thorough investigation and user reports, it appears that the "Fan Control" software has been flagged as "RiskTool.MSIL.cqtf" by Jiangmin antivirus. Notably, some users are experiencing issues with Windows Defender blocking the software after a recent update.

**Comment by Rem0o** [comment]
I have no idea what RiskTool.MSIL.cqtf is, but it sounds like a false positive.

**Comment by Aggnog** [comment]
Thorough investigation lmao, even VLC gets flagged for this exact thing.

**Comment by sheitmaster** [comment]
Might be a wrong flag, but then again it might not.
I just got a trojan from FanControls own updater.exe..

Wacatac.B!ml Trojan.
Windows defender stopped it immediately as i'm up to date, but if it didn't i would potentially have lost everything on this pc and file server.

From now on i'm not updating FanControl. it stays the way it is.

**Reply by Rem0o** [reply]
It's a false flag. Happens whenever the binaries changes for the updater.   I recently updated my compiler, so the binaries changed, and it trips defender.   I sent for analysis, it should get cleared soon if its not already done.
